src/sv_move.c

```diff
@@ -313,7 +313,7 @@ void SV_NewChaseDir (edict_t *actor, edict_t *enemy, float dist)
 	}
 
 	// try other directions
-	if ( ((rand()&3) & 1) ||  abs(deltay)>abs(deltax))
+	if ( ((rand()&3) & 1) ||  fabsf(deltay)>fabsf(deltax))
```

Comment:
Thanks, let's just go for `fabs`, it will be enough to quiet warning.

Reply:
Ah I didn't know `fabs()` was enough. Updated.